### DISCUSSION TITLE
gnina-torch on Windows 10

### DISCUSSION THREAD

**Discussion by MachineGUN001** [RMeli/gnina-torch#55] [Q&A] [answered]
hi,

Will this script work fully in windows 10?
many thanks

Sh-Y

**Comment by RMeli** [comment, marked as answer]
Hello.
I don't know. I never used Windows, and CI only tests on Linux.

I think your best shot would be to use [WSL](https://learn.microsoft.com/en-us/windows/wsl/) on Windows.

**Reply by MachineGUN001** [reply]
thanks!
